src/components/projects.tsx

```
@@ -94,18 +99,18 @@ export default function ProjectPage({ params }: { params: { id: string } }) {
+          goal={Number(project.goal)}
+          daysLeft={project.daysLeft!}
           donations={984}
         />
         <PublisherCard
-          name={project.publisher.name}
-          description={project.publisher.description}
-          totalProjects={project.publisher.totalProjects}
-          totalFundsRaised={project.publisher.totalFundsRaised}
-          trustScore={project.publisher.trustScore}
-          yearFounded={project.publisher.yearFounded}
+          name={author?.name!}
```

ESLint

[error] Optional chain expressions can return undefined by design - using a non-null assertion is unsafe and wrong.

(ts/no-non-null-asserted-optional-chain)